core/src/display_object/edit_text.rs

```diff
@@ -3129,6 +3144,11 @@ impl<'gc> TInteractiveObject<'gc> for EditText<'gc> {
             self.set_selection(None, context.gc());
         }
 
+        if !focused {
```

Comment:
It barely matters performance-wise, but would it make sense to move the check above to be here as:

```rs
            let is_avm1 = !self.movie().is_action_script_3();
            if is_avm1 {
                self.set_selection(None, context.gc());
            }
```